norse/torch/functional/tsodyks_makram.py

```diff
@@ -0,0 +1,36 @@
+from typing import NamedTuple, Tuple
+import torch
+
+
+class TMState(NamedTuple):
+    u: torch.Tensor
+    x: torch.Tensor
+
+
+class TMParameters(NamedTuple):
+    tau_f_inv: torch.Tensor = 1 / 50.0
+    tau_s_inv: torch.Tensor = 1 / 20.0
+    tau_d_inv: torch.Tensor = 1 / 750.0
+    U: torch.Tensor = 0.45  # STD use STF 0.15
```

Comment:
I would maybe add a comment describing the parameter or use a longer variable name.